### PR TITLE
fan page part 4

### PR DESCRIPTION
**Pull request by leyadotnet**
It is good to keep CSS externally for maintenance and to keep clean code.

### CODE REVIEW

**Comment by brianandreasen1983** [comment]
Hi Leya great colorful site you have going on here nice job!

File: index.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        <h2><a href="http://www.myrecipes.com/breakfast-and-brunch-recipes">Breakfast Recipe</a></h2>
+        <ul>
+            <li>cereal</li>
+        </ul>
+        <h2><a href="http://www.myrecipes.com/recipe/chicken-rice-parmesan-skillet">Lunch Recipe</a></h2>
+        <ul>
+            <li>Lasagna</li>
+        </ul>
+        <a href="Images/fan-page.jpg"><img src="Images/fan-page.jpg" alt="thumbnail" height= 50 width= 50></a>
+        <h2><a href="http://www.myrecipes.com/how-to/video/how-to-make-skillet-chicken-roasted-potatoes">Dinner Recipe</a></h2>
```
Comment (mbMosman) [inline index.html]: You should add at least some content under each of these additional headings.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <title>Food Blog</title>
+    <meta charset="utf-8" />
+    <link rel="stylesheet" href="style.css">
+    <link rel="icon" href="myfav.ico" type="image/x-icon">
```
Comment (mbMosman) [inline index.html]: The favicon is still missing from the upload.  You should add it to your Images folder.

File: style.css [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    border-color: #000033;
+    
+    
+}
+
+h1 {
+    color: blueviolet;
+    text-transform: uppercase;
+    font-family: Georgia, "Times New Roman", serif;
+    background-image: url(https://static.pexels.com/photos/5938/food-salad-healthy-lunch-medium.jpg);
```
Comment (mbMosman) [inline style.css]: You shouldn't link to images on other sites like this.  In the web world, this is generally considered "stealing" as it uses the bandwidth to/from their site to put the image on your page.  If it's an image they're ok with you using, it is a better practice to just save the image file and add it to your project folder so it is hosted from their site.  Then give them credit for the image in the footer or directly under the image itself.

**Comment by mbMosman** [comment]
Good work!

**Pull request closed without merging**